external_zstd.go

```diff
@@ -0,0 +1,7 @@
+// +build external_libzstd
```

Comment:
Can you run `go fmt .` on this file using Go 1.17? I believe it should add a separate line with the "new style" build directives, which look like `//go:build` ; See https://golang.org/doc/go1.17#build-lines

(this may imply that CI in this repository is missing a check? Or maybe is running checks with an older version of Go)

Reply:
done